### PR TITLE
Samantha Leal

### PR DESCRIPTION
**Pull request by sammyjoyce26**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [ ] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [ ] The webpage links to another page (multi-page website): 10 points
- [ ] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [ ] The title is centered in front of a background image: 10 points
- [ ] The body images are sized and formatted according to the wire frame: 10 points
- [ ] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [ ] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [ ] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [ ] Responsive navbar: 20 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [ ] Matched layout of original site: 20 points
- [ ] Matched font of original site: 10 points
- [ ] Properly commented code: 10 points
- [ ] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 01week/bio/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<div id="wrapper">
+
+	<div id="header">
+      <img src="https://unsplash.com/?photo=21ae7tGpJtE"
+		<div id="nav"><a href="index.html">Home</a><br></br><a href="#">About Me</a><br></br>
+      <a href="#">Contact Me</a<br></br>
```
Comment (egarcia1868) [inline 01week/bio/index.html:14]: You need an > at the end of your closing anchor tag on line 14.  Also, line breaks don't need a closing tag, so you can remove the /br tags from lines 13 and 14.  You should also split up the different tags into separate lines so they're easier to read.

File: 01week/bio/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<div id="wrapper">
+
+	<div id="header">
+      <img src="https://unsplash.com/?photo=21ae7tGpJtE"
```
Comment (egarcia1868) [inline 01week/bio/index.html:12]: You need to end line 12 with a >

File: 01week/bio/index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        		<p>This is just a fraction of me!</p>
+            <h3>
+				<ul style="margin:10px 20px 10px 20px;">
+					<li>Hiking the Greenbelt</li>
+					<li>Camping</li>
+					<li>Playing with my dog, Tilley</li>
+					<li>Reading</li>
+				</ul>
+      </h3>
+			</div>
```
Comment (egarcia1868) [inline 01week/bio/index.html:35]: You have a few tags left open.  I think there are 4 div tags left open, as well as your body and html.

File: 01week/bio/css/style.css [hunk 4]
```diff
@@ -0,0 +1 @@
+  <img src="https://unsplash.com/?photo=21ae7tGpJtE"/>
```
Comment (egarcia1868) [inline 01week/bio/css/style.css:1]: This is HTML coding, not css.

File: 01week/resume/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+  <html>
+  <head>
+    <title>Resume</title>
+  </head>
+  <link type="text/css" rel="stylesheet" href="stylesheet.css"/>
+  <body style="margin-top:80px; margin-left:80px; margin-right:80px;
+  margin-bottom:80px;">
```
Comment (egarcia1868) [inline 01week/resume/index.html:8]: If you link this html file to a css file, you can do away with the all these properties and values (i.e. - margin-top:80px) in your html file, making it easier to read.

File: 01week/resume/index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    <h3 style="font-family:Times; color:green;
+    text-align:center;font-size:36px;"><u>Skills</u></h3>
+    <ul>
+      <li style="font-size:24px">HTML</li>
+      <li style="font-size:24px">CSS</li>
+      <li style="font-size:24px">JavaScript</li>
+    </ul>
+    <h3 style="font-family:Times; color:green;
+    text-align:center;font-size:36px;"><u>Education</u></h3>
+    <tb>
```
Comment (egarcia1868) [inline 01week/resume/index.html:28]: I'm not familiar with a tb tag, but if it needs a closing tag it is missing it.

File: 01week/resume/index.html [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+      <thead>
```
Comment (egarcia1868) [inline 01week/resume/index.html:29]: There is no closing thead tag

File: 01week/bio/index.html [hunk 8]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
```
Comment (egarcia1868) [inline 01week/bio/index.html:5]: to link your bio stylesheet to this html, I believe you need to change the "style.css" to "css/style.css"

File: 01week/bio/index.html [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+		<div id="left-column">
+			<div id="logo">
+			<h1>Samantha Leal</h1>
+			</div>
+			<div class="box">
+      </h2>
+        		<h1>Who am I?</h1>
+        		<p>This is just a fraction of me!</p>
+            <h3>
+				<ul style="margin:10px 20px 10px 20px;">
```
Comment (egarcia1868) [inline 01week/bio/index.html:28]: If you link this html file to your bio css file you can do away with the margin property and value in your html.  Just make a selector in your css with those properties and values for your html to refer to.

File: 01week/resume/index.html [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+        <tr>
```
Comment (egarcia1868) [inline 01week/resume/index.html:30]: there is no closing tr tag

File: 01week/resume/index.html [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+      <td style="font-size:24px;">McCallum High School- Honors Diploma</td>
+      <br></br>
+      <td style="font-size:24px;">Texas State University- Bachelor of Economics</td>
+      <br></br>
+      <td style="font-size:24px;">Austin Coding Academy
+
```
Comment (egarcia1868) [inline 01week/resume/index.html:36]: no closing body tag

File: 01week/resume/index.html [hunk 12]
```diff
@@ -23,7 +23,7 @@
       <li>Store Manager</li>
       <li>Student Director</li>
       <li>Salon Manager</li>
-      <a href=>Resume</a>
```
Comment (sammyjoyce26) [inline 01week/resume/index.html]: Is this an image link or a website link?

**Pull request closed without merging**